app/code/community/Studioforty9/Recaptcha/Model/Source/Version.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+final class Studioforty9_Recaptcha_Model_Source_Version
+{
+    /**
+     * Options getter
+     *
+     * @return array
+     */
+    public function toOptionArray()
+    {
+        return array(
+            array('value' => 2, 'label'=>Mage::helper('studioforty9_recaptcha')->__('v2')),
+            array('value' => 3, 'label'=>Mage::helper('studioforty9_recaptcha')->__('v3')),
+        );
```

Comment:
```suggestion
        $result = [];
        foreach ($this->toArray() as $value => $label) {
            $result[] = ['value' => $value, 'label' => $label];
        }
        return $result;
```
Non mi piace molto avere la lista dei valori ripetuta due volte. Preferirei implementare uno dei due metodi in funzione dell'altro

Reply:
Ok modifico.

Reply:
Fatto.